examples/qtmegachatapi/MainWindow.cpp

```diff
@@ -863,7 +895,7 @@ void MainWindow::updateLocalChatListItem(MegaChatListItem *item)
     {
         const megachat::MegaChatListItem *auxItem = itItem->second;
         mLocalChatListItems.erase(itItem);
-        addLocalChatListItem(item);
+        addOrUpdateLocalChatListItem(item);
```

Comment:
It would be possible remove the code of this function `MainWindow::updateLocalChatListItem` and call directly to `addOrUpdateLocalChatListItem(item);`